content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,571 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Kubernetes
 
-This pack defines the properties that will be used by Spectro Cloud to bring up the Kubernetes cluster. Most of the Kubernetes hardening standards recommended by the Center for Internet Security is enabled by default.
+The Kubernetes pack defines the properties that Spectro Cloud uses to deploy Kubernetes clusters. Most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends are enabled by default. The Kubernetes pack supports most [cloud and data center infrastructure providers](/clusters).
```

Comment:
```suggestion
The Kubernetes pack defines the default properties that we use to deploy Kubernetes clusters. Most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends are enabled by default. The Kubernetes pack supports most [cloud and data center infrastructure providers](/clusters).
```
This is a good spot for us to use we. Whenever possible, let's use "we" otherwise it can feel like we are creating distance between ourselves and docs.